portfolio.html

```diff
@@ -118,11 +118,11 @@ <h1 class="pl3 pr3 f1 w-100 tc tl-ns dwyl-dark-gray">Portfolio</h1>
     <section class="bg-white pt5 pb6">
       <section class="mw8 center ph3">
         <h2 class="pl3 pr3 f1 mt0 w-100 tc tl-ns dwyl-dark-gray">Open Source</h2>
-        <p class="pl3 pr3 f4 b ph0-m dwyl-dark-gray pb3">We also create and maintain over 200 open source repositories on <a href="https://github.com/dwyl" class="dwyl-mint" >GitHub</a>, check out some of our most popular ones below:</p>
+        <p class="pl3 pr3 f4 b ph0-m dwyl-dark-gray pb3">We also create and maintain over 200 open source repositories on <a href="https://github.com/dwyl" class="dwyl-mint-dark" >GitHub</a>, check out some of our most popular ones below:</p>
```

Comment:
no need to update the color hex for `dwyl-mint` if we are changing the class name to `dwyl-mint-dark`. 💭